api/test/data/import/sample-package/manifest.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: v1beta1
+###########################################################
+# Import Package
+#
+# Pre-Req: Project already exists
+# 
+# POST /api/v1/import?project=<project-name>
+# @package.zip
+#   * manifest.yaml
+#   * api/
+#   * resources/
+###########################################################
+
+tasks:
+  # create a service
+  - name: "Create service"
+    type: api
+    action: "keptn-api-v1-create-service" # pre-defined endpoint types (we only support a few)
+    payload: "api/create-service.json"      # JSON file which contains the not-validated payload 
+    context:                         # context available for templating in payload
+      service: "my-service-name"            # --> {{ .payloadContext.service }}
+
+  # create a secret which we will later use in webhook
+  - name: "Create secret"
+    type: api
+    payload: "api/create-secret.json"
+    action: "keptn-api-v1-uniform-create-secret"
+
+  # Create a subscription
+  - name: "Create webhook subscription"
+    type: api
+    id: create-subscription
+    payload: "api/create-subscription.json"
+    action: "keptn-api-v1-uniform-create-webhook-subscription"
+    # internally this figures out the current webhook integration id
+    context: # -> goes into "api/create-subscription.json"
+      event: "sh.keptn.event.evaluation.triggered"
+      project: "{{ .inputs.project }}"
```

I think we should use the same parentheses for the templating as in the other files.
```suggestion
      project: "[[ .inputs.project ]]"
```